Source/engine.h

```diff
@@ -60,6 +60,26 @@ enum direction : uint8_t {
 struct Point {
 	int x;
 	int y;
+
+	Point operator +(const Point &other)

```

Comment:
From http://en.cppreference.com/w/cpp/language/operators:

> Binary operators are typically implemented as non-members to maintain symmetry

Reply:
It does make it look nicer, even though this struct is trivial.